cfme/cloud/instance/__init__.py

```diff
@@ -1,79 +1,72 @@
-mon_btn = partial(tb.select, 'Monitoring')
-
-tree_inst_by_prov = partial(accordion.tree, "Instances by Provider")
-tree_instances = partial(accordion.tree, "Instances")
-tree_image_by_prov = partial(accordion.tree, "Images by Provider")
-tree_images = partial(accordion.tree, "Images")
-
-list_page = Region(title='Instances')
-
-policy_page = Region(
-    locators={
-        'policy_tree': Tree('//div[@class="containerTableStyle"]/table')
-    })
-
-image_select_form = Form(
-    fields=[
-        ('template_table', Table('//div[@id="pre_prov_div"]//table')),
-        ('cancel_button', form_buttons.cancel)
-    ]
-)
+from utils.wait import wait_for
 
-manage_policies_tree = CheckboxTree("//div[@id='protect_treebox']/ul")
 
-list_table = PagedTable(table_locator="//div[@id='list_grid']//table")
+class CloudInstanceView(BaseLoggedInPage):
+    title = Text('#explorer_title_text')
 
-match_page = partial(match_location, controller='vm_cloud', title='Instances')
+    reload = Button(title='Reload current display')
+    configuration = Dropdown('Configuration')
+    lifecycle = Dropdown('Lifecycle')
+    policy = Dropdown('Policy')
+    power = Dropdown('Power Operations')  # title on instance 'all' pages
+    monitoring = Dropdown('Monitoring')
 
+    views = View.nested(ItemsToolBarViewSelector)
 
-def details_page_check(name, provider):
-    title_match = match_page(summary='Instance "{}"'.format(name))
-    if title_match:
-        # Also check provider
-        try:
-            prov_match = InfoBlock.text('Relationships', 'Cloud Provider') == provider.name
-            return title_match and prov_match
-        except BlockTypeUnknown:
-            # Default to false since we can't identify which provider the image belongs to
-            return False
+    paginator = PaginationPane()
 
+    instance_table = Table("//div[@id='list_grid']//table")
 
-class CloudInstanceTimelinesView(TimelinesView, BaseLoggedInPage):
     @property
-    def is_displayed(self):
-        return self.logged_in_as_current_user and \
-            self.navigation.currently_selected == ['Compute', 'Clouds', 'Instances'] and \
-            super(TimelinesView, self).is_displayed
-
-
+    def in_cloud_instance(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ['Compute', 'Clouds', 'Instances']
+        )
+
+    @View.nested
+    class instances_by_provider(Accordion):  # noqa
+        ACCORDION_NAME = 'Instances by Provider'
+        tree = ManageIQTree()
+
+    @View.nested
+    class images_by_provider(Accordion):  # noqa
+        ACCORDION_NAME = 'Images by Provider'
+        tree = ManageIQTree()
+
+    @View.nested
+    class instances(Accordion):  # noqa
+        ACCORDION_NAME = 'Instances'
+        tree = ManageIQTree()
+
+    @View.nested
+    class images(Accordion):  # noqa
+        ACCORDION_NAME = 'Images'
+        tree = ManageIQTree()
+
+
+@VM.register_for_provider_type('cloud')
```

Comment:
REQUIRED: these things were removed lately by psav during transition to entrypoints. Please make sure you correctly merged with PR #4500.